README.md

```diff
@@ -7,3 +7,29 @@ Allow Journal Managers and (optionally) Authors to deposit articles via the SWOR
 This plugin is best installed using the Plugin Gallery from within Open Journal Systems.
 Log in as Administrator, navigate to `Settings` > `Website` > `Plugins`, and open the "Plugin Gallery"
 tab. Find the Sword plugin listed there and use the `Install` button to install it.
+
+## Configuration
+
+### General Settings
+- After successful installation, you  have to **enable** the plugin in both generic and import export plugin section.
```

Comment:
I don't think it's possible to enable/disable the import/export plugin part of this, is it? All import/export plugins should always be checked "enabled" and the checkbox shouldn't be clickable.

Reply:
> I don't think it's possible to enable/disable the import/export plugin part of this, is it?
yes, I corrected that, wanted to be sure. But sure, it may be confusing , therefore corrected that.